notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_check_alignment_against_human_preference_data.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Non-blocking: These are some random reddit examples we use for testing, wondering if we can find 10 high quality examples. I can take a look around as well.</p><p> </p><p>QA may be better as well, less reading required. Also easier to show alignment since we can have clearly incorrect cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2659/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
There are these examples from the documentation: https://cloud.google.com/vertex-ai/docs/generative-ai/models/side-by-side-eval#example_of_evaluation_dataset